### PR TITLE
Add new voting system

### PR DESCRIPTION
**Pull request by avibn**
Added judging :mhmru:.
Let me know if I forgot to change anything back lol

### CODE REVIEW

File: TWT/apps/timathon/views/judge/judge.py [hunk 1]
```diff
@@ -0,0 +1,76 @@
...
+                    avatar_url = user.get_avatar_url()
+                    if avatar_url.endswith("None.png"):
+                        random = randint(0, 4)
+                        avatar_url = (
+                            f"https://cdn.discordapp.com/embed/avatars/{random}.png"
+                        )
+                    new_member["avatar_url"] = avatar_url
+                    new_member["username"] = user.extra_data["username"]
+                    new_member["discriminator"] = user.extra_data["discriminator"]
+                discord_members.append(new_member)
```
Comment (mohamed040406) [inline TWT/apps/timathon/views/judge/judge.py:66]: shouldn't this also go inside the `else` block? 👀 
if the user is not found (which shouldn't happen anyway) it shouldn't add any user to the `discord_members` list

**Review by mohamed040406 — CHANGES_REQUESTED** [review]

**Review by mohamed040406 — APPROVED** [review]

**Review by Correct-skill — APPROVED** [review]
hopefully this is working lol :p

**Comment by avibn** [comment]
lol I'm hoping too :lamo:

**Comment by Springhunk** [comment]
When you talk through github pull request comments 😳😳

**Comment by avibn** [comment]
- Migrating bootstrap caused some ui issues, so most of the commits are slight UI fixes.
- Changed up the homepage a bit.

**Review by Correct-skill — APPROVED** [review]
Tested... works fine ui looks better :smilecat:

**Comment by avibn** [comment]
Todo:
- [ ] Remove last criterion

and then we should be done for now

**Pull request merged**